tests/cql/CqlListOperatorsTest.xml

```diff
@@ -333,7 +353,7 @@
 		</test>
 		<test name="IncludesNullLeft">
 			<expression>null includes {2}</expression>
-			<output>false</output>
+			<output>null</output>
```

Comment:
CQL-to-ELM translates this as `(null as List<Any) includes {2}`, and according to the spec:
> For the list overload, if either argument is null, the result is null.